migrations/capcons/capabilitymigration.go

```diff
@@ -37,6 +37,16 @@ type CapabilityMigrationReporter interface {
 		accountAddress common.Address,
 		addressPath interpreter.AddressPath,
 	)
+	MissingBorrowType(
+		accountAddress common.Address,
+		addressPath interpreter.AddressPath,
+	)
+	IssuedStorageCapability(
+		accountAddress common.Address,
+		addressPath interpreter.AddressPath,
+		borrowType *interpreter.ReferenceStaticType,
+		capabilityID interpreter.UInt64Value,
+	)
```

Comment:
Maybe move these into a separate interface, this reporter is for the capability value migration, whereas the new functions are for the capabilities issuing function, a related, but different "migration"

Reply:
Refactored in https://github.com/onflow/cadence/pull/3514/commits/97ddc3129f3cb6a6ed90f24887f77c21042ddcd1